_episodes/04-changes.md

```diff
@@ -236,207 +221,182 @@ Changes not staged for commit:
- Cold and dry, but everything is my favorite color
-+The two moons may be a problem for Wolfman
+diff --git a/readme.md b/readme.md
+index 46d72e3..f400cc2 100644
+--- a/readme.md
++++ b/readme.md
+@@ -4,3 +4,5 @@ The `imagecollection.csv` file contains metadata for two objects:
+
+ - an Image object
+ - a Collection object that contains the Image
++
++It has no errors, and will validate against the [metadata-ci script](https://github.com/ucsblibrary/metadata-ci)!
 ~~~
 {: .output}
 
-The output is cryptic because
-it is actually a series of commands for tools like editors and `patch`
-telling them how to reconstruct one file given the other.
-If we break it down into pieces:
-
-1.  The first line tells us that Git is producing output similar to the Unix `diff` command
-    comparing the old and new versions of the file.
-2.  The second line tells exactly which versions of the file
-    Git is comparing;
-    `df0654a` and `315bf3a` are unique computer-generated labels for those versions.
-3.  The third and fourth lines once again show the name of the file being changed.
-4.  The remaining lines are the most interesting, they show us the actual differences
-    and the lines on which they occur.
-    In particular,
-    the `+` marker in the first column shows where we added a line.
+The output is cryptic because it is actually a series of
+machine-readable commands for modifying one version of a file
+(`a/readme.md`) and producing another (`b/readme.md`).  If we break it
+down into pieces:
+
+1.  The first line tells us that Git is producing output similar to
+    the Unix `diff` command comparing the old and new versions of the
+    file.
+2.  The second line tells exactly which versions of the file Git is
+    comparing; `46d72e3` and `f400cc2` are unique identifiers for
+    those versions.
+3.  The third and fourth lines once again show the name of the file
+    being changed.
+4.  The remaining lines are the most interesting, they show us the
+    actual differences and the lines on which they occur.  In
+    particular, the `+` marker in the first column shows where we
+    added a line.  If we had instead removed lines, there would be `-`
+    markers instead.
+
+Because the output of `git diff` is a representation of the difference
+between two files, it’s called (unsurprisingly) a diff.
 
 After reviewing our change, it's time to commit it:
 
 ~~~
-$ git commit -m "Add concerns about effects of Mars' moons on Wolfman"
-$ git status
+$ git commit -m "readme: add line about csv validation"
 ~~~
 {: .bash}
 
-~~~
-On branch master
-Changes not staged for commit:
-  (use "git add <file>..." to update what will be committed)
-  (use "git checkout -- <file>..." to discard changes in working directory)
-
-	modified:   mars.txt
-
-no changes added to commit (use "git add" and/or "git commit -a")
-~~~
-{: .output}
-
-Whoops:
-Git won't commit because we didn't use `git add` first.
-Let's fix that:
+> ## Terminology: diff vs. patch vs. commit
```

Comment:
I'm wondering whether we need to include `patch` in this. `diff` and `commit` I think are super valuable and 'in context' here, but I'm less sure about `patch`. It's just a lot of new. That said, I'm very 50/50 on it..